rotkehlchen/chain/aggregator.py

```diff
@@ -1497,6 +1499,35 @@ def get_evm_manager(
     def is_contract(self, address: ChecksumEvmAddress, chain: SUPPORTED_EVM_CHAINS_TYPE) -> bool:
         return self.get_chain_manager(chain).node_inquirer.get_code(address) != '0x'
 
+    def is_safe_proxy_or_eoa(
+            self,
+            address: ChecksumEvmAddress,
+            chain: SUPPORTED_EVM_CHAINS_TYPE,
+    ) -> bool:
+        """Check if an address is a SAFE contract by checking the getThreshold, VERSION
+        and getChainId methods. We assume that if a contract has the same methods as a safe
+        then it is a safe.
+        """
+        manager: EvmNodeInquirer = self.get_chain_manager(chain).node_inquirer
+        contract = Web3().eth.contract(
+            address=address,
+            abi='[{"inputs":[],"name":"getChainId","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"VERSION","outputs":[{"internalType":"string","name":"","type":"string"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"getThreshold","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"}]',
```

Comment:
Can you put this as a constant somewhere? Also mark it with the appropriate type? When you do you will see that all `internalType` keys can go away.